src/chrome/content/rules/ComputerWoche.de.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="ComputerWoche">
+    <target host="*.computerwoche.de" />
+
+    <rule from="^http://(images|www)\.computerwoche\.de/"
+        to="https://$1.computerwoche.de/" />
```

Comment:
This is patial only because https://www.computerwoche.de/ redirects to http://www.computerwoche.de/

https://images.computerwoche.de/ works.